bazel/external/quiche.BUILD

```diff
@@ -147,6 +151,78 @@ envoy_cc_library(
     deps = ["@envoy//source/extensions/quic_listeners/quiche/platform:spdy_platform_unsafe_arena_impl_lib"],
 )
 
+envoy_cc_library(
+    name = "spdy_core_spdy_alt_svc_wire_format_lib",
+    srcs = ["quiche/spdy/core/spdy_alt_svc_wire_format.cc"],
+    hdrs = ["quiche/spdy/core/spdy_alt_svc_wire_format.h"],
+    copts = quiche_copt,
+    repository = "@envoy",
+    visibility = ["//visibility:public"],
+    deps = [
+        ":spdy_platform",
+    ],
+)
+
+envoy_cc_library(
+    name = "spdy_core_spdy_header_block_lib",
+    srcs = [
+        "quiche/spdy/core/spdy_header_block.cc",
+    ],
+    hdrs = [
+        "quiche/spdy/core/spdy_header_block.h",
+    ],
+    copts = quiche_copt,
+    repository = "@envoy",
+    visibility = ["//visibility:public"],
+    deps = [
+        ":spdy_platform",
+        ":spdy_platform_unsafe_arena_lib",
+    ],
+)
+
+envoy_cc_library(
+    name = "spdy_core_spdy_headers_handler_interface",
+    hdrs = ["quiche/spdy/core/spdy_headers_handler_interface.h"],
+    copts = quiche_copt,
+    repository = "@envoy",
+    visibility = ["//visibility:public"],
+    deps = [":spdy_platform"],
+)
+
+envoy_cc_library(
+    name = "spdy_core_spdy_protocol_lib",
+    hdrs = [
+        "quiche/spdy/core/spdy_bitmasks.h",
+        "quiche/spdy/core/spdy_protocol.h",
+    ],
+    copts = quiche_copt,
+    repository = "@envoy",
+    visibility = ["//visibility:public"],
+    deps = [
+        ":spdy_core_spdy_alt_svc_wire_format_lib",
+        ":spdy_core_spdy_header_block_lib",
+        ":spdy_platform",
+    ],
+)
+
+envoy_cc_test_library(
+    name = "spdy_core_spdy_test_utils_lib",
+    srcs = [
```

Comment:
nit: fold "srcs" and "hdrs" into single lines.

Reply:
done